dist-git-8.0.1-to-8.1.0.patch

```diff
@@ -0,0 +1,51 @@
+From 99f727df0d6e0382beb7960e6c913a163f61bee3 Mon Sep 17 00:00:00 2001
+From: Skyler Clark <skylerjaneclark@gmail.com>
+Date: Mon, 10 Jul 2023 12:22:28 -0400
+Subject: [PATCH] [release-4.13] Prepare for 8.1.0 release
+
+  This commit was generated by hack/pre-release.sh
+
+Signed-off-by: Skyler Clark <skylerjaneclark@gmail.com>
+---
+ Dockerfile       | 2 +-
+ render_templates | 4 ++--
+ 2 files changed, 3 insertions(+), 3 deletions(-)
+
+diff --git a/Dockerfile b/Dockerfile
+index 6a93a28..ddfe25e 100644
+--- a/Dockerfile
++++ b/Dockerfile
+@@ -24,7 +24,7 @@ LABEL com.redhat.openshift.versions="=v4.13"
+ LABEL com.redhat.delivery.backport=false
+ 
+ # This label gets replaced by render_templates, so it is safer to keep it on its own
+-LABEL version="v8.0.1"
++LABEL version="v8.1.0"
+ 
+ # This label maps to the brew build target
+ LABEL com.redhat.component="windows-machine-config-operator-bundle-container"
+diff --git a/render_templates b/render_templates
+index 0c25692..e526ec6 100755
+--- a/render_templates
++++ b/render_templates
+@@ -45,7 +45,7 @@ update_csv() {
+     sed -i "s|REPLACE_IMAGE|${wmco_image_sha}|g" ${csv}
+     # Following command adds the replaces field in the CSV using the previous released version of the operator.
+     # TODO: Uncomment below for next minor release
+-    # sed -i '/version: 8.0.1/i \  replaces: '${wmco}'.v'${previous_version}'' ${csv}
++    # sed -i '/version: 8.1.0/i \  replaces: '${wmco}'.v'${previous_version}'' ${csv}
```

Comment:
When releasing minor versions this line (and the previous_version line) below should be uncommented. And only when releasing a `x.0.0` should both be commented out

Reply:
This is a manual step right now as listed in our [release docs](https://docs.google.com/document/d/1F5TXHgvUcca5Pzsto2gx-eA8Hm-Eb47xxStWU35VgeA/edit#heading=h.f1l61286xjes). It would be great if the pre-release script ensured this for us

Reply:
:+1: can do!